Source/utils/sdl2_to_1_2_backports.cpp

```diff
@@ -4,6 +4,10 @@
 
 #include "./console.h"
 
+#ifdef __PSP__
+#include "miniwin/miniwin.h"
+#endif

```

Comment:
Is this still needed?

Reply:
It's needed for the SDL_snprintf redirect to work.